capstone/src/main/webapp/index.html

```diff
@@ -11,5 +13,6 @@ <h2>Welcome to UniViz!</h2>
       <input type="text" id="search" name="search"><br>
       <input type="submit" value="See your colleges!">
     </form>
+    <div id="data"></div>
```

Comment:
just note that the data ```<div>``` will be on the dashboard.html page and not the index.html page

Reply:
Ah ok, when that gets merged into master I'll move the `<div>` there.